back/benchmarks/Main.hs

```diff
@@ -0,0 +1,49 @@
+-- | Module contains all stuff to migrate from AcidState to Postgres.
+module Main
+       (
+         main
+       ) where
+
+import Imports
+
+import Gauge
+import Hasql.Transaction.Sessions (Mode (..))
+
+import Guide.Database.Queries.Update
+import Guide.Database.Queries.Select
+import Guide.Types.Core
+import Guide.Database.Connection
+
+
+main :: IO ()
+main = do
+    conn <- connect
+    defaultMain [databaseBenchmark conn]
+  where
+    databaseBenchmark conn =
+      bgroup
+        "Database"
+        [ bench "select" $ nfIO $
+            runTransactionExceptT conn Read $ selectCategory "category1111"
```

Comment:
Please let's not have such names in code, they look sloppy. "category" or "uid1" would be better.

Reply:
length "category1111" == 12 as Uid needed.